### DISCUSSION TITLE
Emmet completion with arbitrary values

### DISCUSSION THREAD

**Discussion by wenfangdu** [tailwindlabs/tailwindcss#11627] [Help]
`.pl-[20px]` is expanded to `<div class='pl-' 20px=''></div>`, but I want it to be expanded to `<div class='pl-[20px]'></div>`, any suggestions?

**Comment by mybearworld** [comment]
It's not that important for it to work, as you should rarely use these. When you use these, you're more likely to have a tag anyways. (Side note, why are you using 20px as an arbitrary value? Just use pl-5...)

**Reply by wenfangdu** [reply]
@mybearworld We're converting an existing site to tailwind, using `.pl-[20px]` is easier for migration.

**Comment by sushmitg** [comment]
same problem.

![image](https://github.com/user-attachments/assets/5bbd2825-815c-4d68-9781-8355edbfe3c3)

emmet expand abbreviation not working properly
